sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -111,6 +111,24 @@ def test_remote_runner_translation(self):
     remote_runner.job = apiclient.Job(p._options)
     super(DataflowRunner, remote_runner).run(p)
 
+  def test_streaming_create_translation(self):
+    remote_runner = DataflowRunner()
+    self.default_properties.append("--streaming")
+    p = Pipeline(remote_runner, PipelineOptions(self.default_properties))
+    p | ptransform.Create([1])  # pylint: disable=expression-not-assigned
+    remote_runner.job = apiclient.Job(p._options)
+    # Performing configured PTransform overrides here.
+    p.replace_all(DataflowRunner._PTRANSFORM_OVERRIDES)
```

Comment:
This is subtle enough to deserve a comment. (Is there a way instead to use templates or another option to get the job description without submitting it?)